web/src/App.css

```diff
@@ -0,0 +1,679 @@
+@font-face { font-family: 'customFont'; 
+  src: url('/fonts/Poppins.ttf') format('truetype'); font-weight: normal; font-style: normal; }
+*{
+  margin: 0;
+  padding: 0;
+}
+/* #070F12, #003B46, #006770, #008C8B, #00A3A9 */
+html{
+  background-color: #f1f1f1;
+  width: 100vw;
+  font-family: 'customFont';
+  overflow-x: hidden;
+}
+.container{
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  
+}
+nav{
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  color: #017481;
+  background-color: #f2f3f4;
```

Comment:
We can use CSS variables to make maintenance easier. For example, colors could be placed in variables and reused